qt/data/PdfExportDialog.ui

```diff
@@ -170,103 +274,46 @@
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
-    <widget class="QGroupBox" name="groupBoxImage">
-     <property name="title">
-      <string>Image settings:</string>
+    <widget class="QGroupBox" name="groupBoxGeneral">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
      </property>
-     <property name="flat">
-      <bool>false</bool>
+     <property name="minimumSize">
+      <size>
+       <width>0</width>
+       <height>0</height>
+      </size>
      </property>
-     <layout class="QGridLayout" name="gridLayout_2">
-      <property name="leftMargin">
-       <number>4</number>
-      </property>
-      <property name="topMargin">
-       <number>4</number>
-      </property>
-      <property name="rightMargin">
-       <number>4</number>
-      </property>
-      <property name="bottomMargin">
-       <number>4</number>
-      </property>
-      <item row="2" column="1">
-       <widget class="QSpinBox" name="spinBoxDpi">
-        <property name="minimum">
-         <number>50</number>
-        </property>
-        <property name="maximum">
-         <number>1200</number>
-        </property>
-        <property name="singleStep">
-         <number>10</number>
-        </property>
-        <property name="value">
-         <number>300</number>
-        </property>
-       </widget>
-      </item>
-      <item row="4" column="0">
-       <widget class="QLabel" name="labelCompressionQuality">
-        <property name="enabled">
-         <bool>false</bool>
-        </property>
-        <property name="text">
-         <string>Compression quality:</string>
-        </property>
-       </widget>
-      </item>
-      <item row="2" column="0">
-       <widget class="QLabel" name="labelDpi">
-        <property name="text">
-         <string>DPI:</string>
-        </property>
-       </widget>
-      </item>
-      <item row="0" column="1">
-       <widget class="QComboBox" name="comboBoxImageFormat"/>
-      </item>
-      <item row="4" column="1">
-       <widget class="QSpinBox" name="spinBoxCompressionQuality">
-        <property name="maximum">
-         <number>100</number>
-        </property>
-        <property name="value">
-         <number>90</number>
-        </property>
-       </widget>
-      </item>
-      <item row="3" column="1">
-       <widget class="QComboBox" name="comboBoxImageCompression"/>
-      </item>
+     <property name="maximumSize">
+      <size>
+       <width>393</width>
+       <height>16777215</height>
+      </size>
+     </property>
+     <property name="title">
+      <string>General</string>
```

Comment:
The other groupbox titles have a colon, either add it here, or we can remove it from the other titles (and update all translations... ;( )

Reply:
Oh, again, one more colon :-)